src/workloads/tpch/normalized/1/Q22.yml

```diff
@@ -0,0 +1,34 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-query"
+Description: |
+  Run TPC-H query 22 against the normalized schema for scale 1.
+
+Clients:
+  Default:
+    QueryOptions:
+      socketTimeoutMS: -1
+
+Actors:
+- Name: TPCHNormalizedQuery22
+  Type: RunCommand
+  Database: tpch
+  Phases:
+  - LoadConfig:
+      Path: ../../../../phases/tpch/normalized/Q22.yml
+      Key: TPCHNormalizedQuery22
+      Parameters:  # This is a hack, because yml parses these strings into ints before outputting the pipeline.
```

Comment:
Oof what behavior would be preferred here? The transformation might also be happening internal to genny too, we have our own set of types that are interpreted from the yaml.

Reply:
Oof gotcha. Not sure whether that's a yaml thing or a genny thing, or both. I don't suppose some combination of internal single quotes or escapes helps? Feel free to file a TIG ticket if this is a pain! We're going to be thinking a lot about syntax stuff soon anyway.